dotCMS/src/main/java/com/dotcms/filters/RequestTrackingFilter.java

```diff
@@ -0,0 +1,116 @@
+        try {
+            requestTracker = RequestTracker.getInstance();
+            
+            // Check if tracking should be disabled via init parameter
+            String trackingParam = filterConfig.getInitParameter("requestTrackingEnabled");
+            if ("false".equalsIgnoreCase(trackingParam)) {
+                trackingEnabled = false;
+                Logger.info(this, "Request tracking disabled via init parameter");
+            } else {
+                Logger.info(this, "RequestTrackingFilter initialized - unified request tracking enabled");
+            }
+            
+        } catch (Exception e) {
+            Logger.error(this, "Failed to initialize RequestTrackingFilter: " + e.getMessage(), e);
+            throw new ServletException("RequestTrackingFilter initialization failed", e);
+        }
+    }
+    
+    @Override
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) 
+            throws IOException, ServletException {
+        
+        // Only process HTTP requests
+        if (!(request instanceof HttpServletRequest) || !(response instanceof HttpServletResponse)) {
+            chain.doFilter(request, response);
+            return;
+        }
+        
+        HttpServletRequest httpRequest = (HttpServletRequest) request;
+        HttpServletResponse httpResponse = (HttpServletResponse) response;
+        
+        // Skip tracking if disabled
+        if (!trackingEnabled || requestTracker == null) {
+            chain.doFilter(request, response);
+            return;
+        }
+        
+        // Mark request start for tracking
+        requestTracker.requestStarted(httpRequest);
+        
+        try {
+            // Process the request through the filter chain
+            chain.doFilter(request, response);
+            
+        } finally {
+            // Always mark request end, even if an exception occurred
+            try {
+                requestTracker.requestEnded(httpRequest, httpResponse);
+            } catch (Exception e) {
+                // Don't let tracking errors affect the response
+                Logger.debug(this, "Error during request end tracking: " + e.getMessage());
+            }
+        }
+    }
+    
+    @Override
+    public void destroy() {
+        try {
+            Logger.info(this, "RequestTrackingFilter destroyed");
+            
+            // Log final statistics
+            if (requestTracker != null && trackingEnabled) {
+                long totalRequests = requestTracker.getTotalRequests();
+                long activeRequests = requestTracker.getActiveRequests();
+                double averageDuration = requestTracker.getAverageDuration();
+                double errorRate = requestTracker.getErrorRate();
+                
+                Logger.info(this, String.format("Request tracking statistics - Total: %d, Active: %d, Avg Duration: %.2fms, Error Rate: %.2f%%",
```

Comment:
Probably an appender+logger for this on the log4j.xml would be a good idea

Reply:
Could be something to consider for the future,  I don't want to add more complication in this PR,  this message is only reported once in the shutdown process and should not contribute much to log bloat.   We should not be reporting more regular logging of this information on the running server going forward as those should be covered by Prometheus/Opentelemetry rather than needing any additional log.